pkg/authorize/jwt/handler.go

```diff
@@ -77,7 +77,9 @@ func (a *authorizeClusterHandler) ServeHTTP(w http.ResponseWriter, req *http.Req
 		}
 
 		if scerr, ok := err.(statusCodeErr); ok {
-			log.Printf("error: unable to authorize request: %v", scerr)
+			if scerr.HTTPStatusCode() >= http.StatusInternalServerError {
+				log.Printf("error: unable to authorize request: %v", scerr)
```

Comment:
Hmmm we should *really* be using leveled logging imo and log all failures at least as infos and all 5xxs as errors. For now this is ok

Reply:
There are different schools out there, the Go ecosystem mostly circles around Peter's principles: https://peter.bourgon.org/blog/2016/02/07/logging-v-instrumentation.html

But k8s goes a different route obviously.